<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimulDAQDlg</class>
 <widget class="QDialog" name="SimulDAQDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>217</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simulated DAQ Setup</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>221</x>
     <y>170</y>
     <width>160</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="InputFileE">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>StdpcIn1.dat</string>
   </property>
  </widget>
  <widget class="QLabel" name="InputFileL">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>45</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Input File</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="OutputFileE">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>StdpCOut1.dat</string>
   </property>
  </widget>
  <widget class="QLabel" name="OutputFileL">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>53</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Output File</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="inChannelE">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QLabel" name="inChannelL">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>66</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string># in Channels</string>
   </property>
  </widget>
  <widget class="QLabel" name="outChannelL">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>73</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string># out Channels</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="outChannelE">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="outDtE">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>130</y>
     <width>48</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="outDtL">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>130</y>
     <width>47</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Output dt</string>
   </property>
  </widget>
  <widget class="QLabel" name="outDtU">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>130</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ms</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="inTFacE">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>100</y>
     <width>48</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="inTFacU">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>100</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>s</string>
   </property>
  </widget>
  <widget class="QLabel" name="InTFacL">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>100</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Input t conversion  1 =</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SimulDAQDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SimulDAQDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
